core/src/test/java/tech/pegasys/eth2signer/core/multikey/DirectoryBackedArtifactSignerProviderTest.java

```diff
@@ -208,13 +216,15 @@ void signerIdentifiersReturnedForMetadataFile() throws IOException {
     createFileInConfigsDirectory(PUBLIC_KEY1);
     when(signerParser.parse(any())).thenReturn(artifactSigner);
 
+    signerProvider.loadSigners();
```

Comment:
A test that shows the loadSigners loads the signers into cache would be good

Reply:
The cache is irrelevant - if you load files, and ask for public keys - you should get back the same number - do you _care_ how/where they are stored?

Reply:
No I don't care how they are stored. But there is a function loadSigners that does something we should test that it does what we expect.